src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -21,45 +34,334 @@ const ActorInit Bg_Spot15_Rrbox_InitVars = {
     (ActorFunc)BgSpot15Rrbox_Update,
     (ActorFunc)BgSpot15Rrbox_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B3960.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot15_Rrbox/func_808B39E8.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+Vec3f D_808B45C4[] = {
+    { 770.0f, 1490.0f, -299.0f },
+    { 770.0f, 1550.0f, -299.0f },
+};
+
+Vec3f D_808B45DC[] = { { 29.99f, 0.01f, -29.99f },
+                       { -29.99f, 0.01f, -29.99f },
+                       { -29.99f, 0.01f, 29.99f },
+                       { 29.99f, 0.01f, 29.99f },
+                       { 0.0f, 0.01f, 0.0f } };
```

Comment:
We leave trailing commas in cases like these to help with the vertical formatting (from clang-format)
```suggestion
                       { 0.0f, 0.01f, 0.0f }, };
```